gtsam/hybrid/HybridGaussianFactor.h

```diff
@@ -89,32 +89,49 @@ class GTSAM_EXPORT HybridGaussianFactor : public HybridFactor {
   HybridGaussianFactor() = default;
 
   /**
-   * @brief Construct a new hybrid Gaussian factor.
+   * @brief Construct a new HybridGaussianFactor on a single discrete key,
```

Comment:
I'm wondering if this should be after the `GaussianFactorValuePair` constructor and specify that the extra error `E_m` is 0.0 in this case?

Reply:
My PR is about doing the opposite of that :-) Go from easy to harder, guiding the user. In fact, the more complex constructors are only used once in the entire code (before I added unit tests).

Reply:
Okay I see that. Awesome.